### PR TITLE
Implemented bunyan mode with level()

### PR DESCRIPTION
**Pull request by mcollina**
Fixes https://github.com/pinojs/pino/issues/276.

We can possibly add `levels()` support on top of this, if it's needed.

cc @yunong

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -109,6 +109,17 @@ log.info('this will be written to /tmp/debug.stream.out and /tmp/info.stream.out
 log.fatal('this will be written to /tmp/debug.stream.out, /tmp/info.stream.out and /tmp/fatal.stream.out')
 ```
 
+### pinoms.level set accessor
+
+You can set the level to _all streams_ by changing the level property.
+It accepts the same parameters of pino.
```
Comment (jsumners) [inline Readme.md]: > ... parameters ~of~ as Pino.

File: package.json [hunk 2]
```diff
@@ -4,9 +4,9 @@
   "description": "A wrapper for the Pino logger that provides Bunyan's multipe destination stream API",
   "main": "index.js",
   "scripts": {
-    "test": "standard && tap --no-cov test/*.js",
+    "test": "standard && tap --no-cov test/*test.js",
```
Comment (jsumners) [inline package.json]: Nit: `*.test.js`

File: Readme.md (hunk 1 above)
Comment (mcollina) [inline Readme.md]: I'm not completely convinced of the current implementation of this feature and its interaction with child loggers.

Comment (jsumners) [inline Readme.md]: This is just a grammar correction change.

Comment (mcollina) [inline Readme.md]: yes I know. I was just pointing it out that I'm not sure of the implementation of this behavior :).

**Comment by jsumners** [comment]
I'm approving this with the caveat that I don't like the way `.level('info')` would set the level of _all_ streams. One of the benefits to multiple streams is being able to have an "info" and a "debug" stream at the same time. But this poor way of doing this is the way Bunyan does it, and that's what has been requested.

**Comment by mcollina** [comment]
@jsumners the implementation of the level-changing logic is now improved, and it fully respect the child-logger encapsulation.

File: package.json [hunk 3]
```diff
@@ -28,12 +28,12 @@
...
     "standard": "^10.0.2",
-    "pino": "^4.6.0",
-    "tap": "^10.3.3"
+    "tap": "^10.7.1"
   },
   "dependencies": {
-    "pino": "^4.6.0"
+    "pino": "^4.7.1"
   },
   "peerDependencies": {
```
Comment (jsumners) [inline package.json]: If it is going to be a hard dependency, do we need the peer?

**Comment by jsumners** [comment]
LGTM. Merge whenever you are ready.

**Pull request merged**